frontend/apps/reader/modules/readerfont.lua

```diff
@@ -71,24 +74,18 @@ end
 
 function ReaderFont:setupTouchZones()
     if Device:isTouchDevice() then
-        self.ui:registerTouchZones({
```

Comment:
Yes, TouchZone is the referred way to handle touch related event registration. It's more efficient and handles screen resizes automatically for you. Most importantly, it supports overriding so you can define priorities between multiple overlapping zones.